### DISCUSSION TITLE
How to change connection string of a Oqtane installation

### DISCUSSION THREAD

**Discussion by Scippy** [oqtane/oqtane.framework#1049] [General]
Hi, how I can change the connection string of an installed Oqtane.
I need to move it from a PC to another where I've different SQL access mode and looking at web.config I can't find any connection string.
Thanks

**Comment by PavelVsl** [comment]
It is in appsettings.json file.

**Comment by shivampayasi01** [comment]
> Hi, how I can change the connection string of an installed Oqtane. I need to move it from a PC to another where I've different SQL access mode and looking at web.config I can't find any connection string. Thanks

I have also faced the same issue but I got connectionstring in appsettings.jason in oqtane framework /server/appsettings.jason 
After updating here I was getting old data so i have updated the connection string from new db with the query 
 **update [dbo].[Tenant] set DBConnectionstring ='your connection string here ' 
where TenantId=1**
I dont Know that its a perfect way or not Now my connection is working fine

**Comment by leigh-pointer** [comment]
@shivampayasi01 the connection string is also stored in the Tenants table.

**Reply by shivampayasi01** [reply]
yes I was also surprised

**Reply by rchibnik** [reply]
I wrestled with this for half a day; Didnt think the database conn string would be stored in the database the conn string pointed to, but ok; easy enough change.

**Reply by sbwalker** [reply]
@rchibnik the connection  strings have all been relocated to appsettings - see https://www.oqtane.org/blog/!/57/connection-string-management

**Comment by sbwalker** [comment]
@Scippy this item was referenced in #2567 and will be addressed in the next release.

**Reply by sbwalker** [reply]
This change has been implemented - see my comment here https://github.com/oqtane/oqtane.framework/issues/2567#issuecomment-1442494640
